srcpkgs/easyeffects/template

```diff
@@ -10,12 +10,12 @@ makedepends="gtkmm4-devel pipewire-devel zita-convolver-devel lilv-devel
  soundtouch-devel speexdsp-devel json-c++ tbb-devel libadwaita-devel fmt-devel
  gsl-devel ladspa-sdk"
 short_desc="Sound effects for systems using PipeWire"
-maintainer="zenobit <zenobit@disroot.org>"
```

Comment:
don't change this

Reply:
Ok, Thank you, I should then create another branch and follow through correctly?

Reply:
no, amend the commit and force-push

Reply:
Thank you, Got it all figured out and pushed.

Reply:
what i meant was:
```
git add srcpkgs/easyeffects/template
git commit --amend
git push --force
```
looks like you made a second commit instead. to fix that:
```
git reset --soft HEAD^
git commit --amend
git push --force
```

Reply:
I am now lost, I thought I had it correct, Yet after following the instructions it pushed some other stuff in with it?

Reply:
github showed two commits so I'm not sure either